legacy_plugins/whitelabelling_bar/src/dp_whitelabelling_bar/components/WhiteLabellingBar.tsx

```diff
@@ -0,0 +1,70 @@
+// (C) 2023 GoodData Corporation
+
+import React, { CSSProperties, useMemo, useRef, useState } from "react";
+import {
+    CustomDashboardWidgetComponent,
+    ICustomWidget,
+    selectIsExport,
+    useDashboardSelector,
+} from "@gooddata/sdk-ui-dashboard";
+import useResizeObserverDefault from "use-resize-observer";
+
+import { DASHBOARD_CONTENT_ROOT_CLASS_NAME } from "../constants.js";
+
+// There are known compatibility issues between CommonJS (CJS) and ECMAScript modules (ESM).
+// In ESM, default exports of CJS modules are wrapped in default properties instead of being exposed directly.
+// https://github.com/microsoft/TypeScript/issues/52086#issuecomment-1385978414
+const useResizeObserver = useResizeObserverDefault as unknown as typeof useResizeObserverDefault.default;
+
+export interface IWhiteLabellingBarWidgetExtras {
+    text?: string;
+    logoUrl?: string;
+    logoAltText?: string;
+}
+
+type WhiteLabellingBarWidgetProps = ICustomWidget & IWhiteLabellingBarWidgetExtras;
+
```

Comment:
@dremileta please add here a brief annotation of what the component does (adds a bar with fixed position at the bottom of the dashboard, the resize observer is used to keep the component at the same width as the dashboard) as requested in the review ticket: https://gooddata.atlassian.net/browse/SPACE-2568?focusedCommentId=3218003